rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -281,16 +281,27 @@ To view the run results and to download an analysis package:
 . If you want details on a result, click a number.
 . In case of errors, click the *Download* icon.
 
-[[publish-automation-exchange]]
-== Publish a Process Automation to Anypoint Exchange
+[[publish-automation]]
+== Publish a Process Automation
 
-Transition the project to the production phase to publish an invokable run configuration to Anypoint Exchange.
+Transition the project to the production phase and publish an invocable run configuration to use the process out of MuleSoft RPA, for example with one of the following products:
+* Anypoint Exchange
+* MuleSoft Composer
+* Salesforce Flow
 
-Projects that have already been used productively can no longer be deleted. Deleted invokable run configurations are not deleted in Anypoint Exchange. Edited and updated invokable run configurations are published as new versions in  Anypoint Exchange.
+Projects that have already been used productively can no longer be deleted. 
```

Comment:
```suggestion
You can't delete projects once they are used in the production phase. 
```

Reply:
Can you say "...that are used in production" instead of "...that are used in the production phase"?